### PR TITLE
82 instr2raw elegant

### LINKED ISSUE #82: instr2Raw.cpp:11: add template parameter by format and...
The puzzle `54-89a87874` from #54 has to be resolved: 

https://github.com/Tea-club/ChaiVM/blob/fb013fba8e396144f245be94b4d64d205f0b866d/src/ChaiVM/utils/instr2Raw.cpp#L11-L11

The puzzle was created by Lev Bagryansky on 19-Feb-24. 

Estimate: 60 minutes,  role: DEV. 

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

### PR DESCRIPTION
**Pull request by levBagryansky**
closes #82

### CODE REVIEW

**Comment by levBagryansky** [comment]
@c71n93 please merge

File: include/frontend/assembler/assembler.hpp [hunk 1]
```diff
@@ -142,17 +142,17 @@ class Assembler final {
         return chai::utils::instr2RawRI(op, func_id, func_id);
     }
     chai::bytecode_t processN(chai::interpreter::Operation op) {
-        return chai::utils::instr2Raw(op, 0, 0);
+        return chai::utils::instr2RawRR(op, 0, 0);
```
Comment (c71n93) [inline include/frontend/assembler/assembler.hpp:145]: @levBagryansky replace it by `instr2RawN` pls

**Review by c71n93 — CHANGES_REQUESTED** [review]
@levBagryansky good job. just one little fix.

File: include/frontend/assembler/assembler.hpp (hunk 1 above)
Comment (levBagryansky) [reply include/frontend/assembler/assembler.hpp]: @c71n93 done

**Review by c71n93 — APPROVED** [review]
@levBagryansky @rultor merge

**Pull request merged**